runtime/userspace/apps/spdm/src/config.rs

```diff
@@ -0,0 +1,96 @@
+// Licensed under the Apache-2.0 license
+
+use spdm_lib::cert_store::MAX_CERT_SLOTS_SUPPORTED;
+use spdm_lib::protocol::SHA384_HASH_SIZE;
+
+pub const MAX_CERT_SIZE: usize = 1024;
+
+pub const SLOT0_DPE_LEAF_CERT_LABEL: [u8; SHA384_HASH_SIZE] = [
```

Comment:
Maybe not for this PR, but we should start collecting these sorts of things in an integrator's doc soon so that people adopting this SDK will know that this is something they will likely need to modify.